test/README.md

```diff
@@ -14,21 +14,35 @@ CONNECTION_PARAMETERS = {
 }
 ```
 
-### Running a single test
-
-Assuming that all dependencies are installed, running a single test is as simple as:
-`python -m pytest test/integ/test_connection.py::test_basic`.
-
 ### Running a suite of tests
 
-We use `tox` to run test suites and other utilities.
+We use `tox` version 4 to run test suites and other utilities.
 
 To run the most important tests, execute:
 
 ```shell
 tox -e "fix_lint,py37{,-pandas,-sso}"
 ```
 
+**NOTE** Some integration tests may be sensitive to the cloud provider of the
+account that the test suite connects to.  By default, when testing locally,
+the "provider" is called `dev`, but you may see some test failures with this
```

Comment:
I'd word this a little differently. `dev` is simply a made up provider that acts as all cloud providers and it's intended so that devs do not need to set `cloud_provider`